package.json

```diff
@@ -9,6 +9,7 @@
     "build": "rimraf dist/ && babel ./ --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
     "clean:build": "rimraf dist/",
     "test": "cross-env NODE_ENV=test nyc --reporter=html mocha --require babel-core/register server/test/index.spec.js --exit && nyc report",
```

Comment:
Remove one of the two test scripts in package.json, retain ```test2``` and remove ```test```. Also rename ```test2``` to ```test```